packages/fxa-settings/src/lib/types.ts

```diff
@@ -75,3 +75,5 @@ export type TotpInfo = {
   secret: string;
   recoveryCodes?: string[];
 };
+
+export type MfaScope = 'test';
```

Comment:
The `MfaScope` type is just 'test' doesn't look right.  I thought it's string that represents of describes the action the user wants to take with the token?